homeassistant/components/bluetooth/active_update_coordinator.py

```diff
@@ -109,6 +111,13 @@ async def _async_poll(self) -> None:
 
         try:
             update = await self._async_poll_data(self._last_service_info)
+        except BleakError as exc:
+            if self.last_poll_successful:
+                self.logger.info(
```

Comment:
```suggestion
                self.logger.error(
```

I think we should still look at error level to match https://github.com/home-assistant/core/blob/b0c093ac70d36bfa3c7225208e751d5330f517b6/homeassistant/helpers/update_coordinator.py#L218